tests/Unit/Domain/CoordinateMaps/Test_Wedge3D.cpp

```diff
@@ -45,32 +45,116 @@ SPECTRE_TEST_CASE("Unit.Domain.CoordinateMaps.Wedge3D.Sphere",
   test_coordinate_map_argument_types<false>(map, test_point1);
```

Comment:
Remove `<false>` (the parameter indicates whether to check the inverse and defaults to true).